### ISSUE TITLE
How to get pending approved airports?

### ISSUE THREAD

**Issue by fabio-alss-freitas** [openAIP/openaip#325]
I would like to filter the airports that are pending approval, is this possible? This is necessary because I want to be notified every time a user searches for an airport that is not in the base and that has not already been entered by someone and is pending.

**Comment by reskume** [comment]
No, this is currently not possible. Airports that are not yet "approved" are only available as an RFC document which is not intended to be queried like this.
